### PR TITLE
Feature/civ 3379 part admit no paid part 2

### PR DESCRIPTION
**Pull request by RosDiaf**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 1]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
 const buildSummarySections = (claim: Claim, claimId: string, lang: string | unknown): SummarySections => {
   const responseType: string = claim.respondent1?.responseType;
   const paymentOption: string = claim.paymentOption;
-  const paidResponse = claim.partialAdmission?.paymentIntention?.paymentOption;
+  const alreadyPaidPartAdmit: string = claim.partialAdmission?.alreadyPaid?.option;
+  const paidResponse: string = claim.partialAdmission?.paymentIntention?.paymentOption;
+
+  const getResponseToClaim = (responseType: string) => {
+    return responseType === ResponseType.FULL_DEFENCE
+      || responseType === ResponseType.FULL_ADMISSION && paymentOption !== PaymentOptionType.IMMEDIATELY
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts]: You can use `claim.isFullDefence()`, `claim.isFullAdmission()` and `claim.isPartialAdmission()` where needed.

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 2]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
+      ? buildYourResponseToClaimSection(claim, claimId, lang)
+      : null;
+  };
+
+  const getResponseToClaimPA = (responseType: string) => {
+    return responseType === ResponseType.PART_ADMISSION
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts]: same here

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 3]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
+      ? buildYourResponseToClaimSection(claim, claimId, lang)
+      : null;
+  };
+
+  const getResponseDetailsSection = () => {
+    return responseType === ResponseType.FULL_DEFENCE || responseType === ResponseType.PART_ADMISSION
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts:44]: same here

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 4]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
+      ? buildYourResponseDetailsSection(claim, claimId, lang)
+      : null;
+  };
+
+  const getFinancialSectionFA = (responseType: string) => {
+    return responseType === ResponseType.FULL_ADMISSION && paymentOption !== PaymentOptionType.IMMEDIATELY
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts]: same here.

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 5]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
+      ? buildYourFinancialSection(claim, claimId, lang)
+      : null;
+  };
+
+  const getFinancialSectionPA = (responseType: string) => {
+    return responseType === ResponseType.PART_ADMISSION && alreadyPaidPartAdmit === YesNo.NO && paidResponse !== PaymentOptionType.IMMEDIATELY
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts]: same here

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 6]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
+      ? buildYourFinancialSection(claim, claimId, lang)
+      : null;
+  };
+
+  const getResponsePaymentSection = () => {
+    return responseType === ResponseType.FULL_ADMISSION
+      || responseType === ResponseType.PART_ADMISSION && alreadyPaidPartAdmit === YesNo.NO
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts:63]: same here

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts [hunk 7]
```diff
@@ -16,23 +16,72 @@ import {buildYourResponsePaymentSection} from './responseSection/buildYourRespon
...
+      ? buildYourResponsePaymentSection(claim, claimId, lang)
+      : null;
+  };
+
+  const getFreeTelephoneMediationSection = () => {
+    return responseType === ResponseType.FULL_DEFENCE
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/checkAnswersService.ts:69]: same here.

File: src/main/services/features/response/checkAnswers/responseSection/buildYourResponsePaymentSection.ts [hunk 8]
```diff
@@ -7,57 +7,73 @@ import {
...
+      break;
+  }
+
+  switch (paymentOption) {
     case PaymentOptionType.IMMEDIATELY:
-      responseSection.summaryList.rows.push(...[
-        summaryRow(t('PAGES.CHECK_YOUR_ANSWER.OWE_MONEY', {lng: getLng(lang)}), t(`COMMON.RESPONSE_TYPE.${claim.respondent1.responseType}`, {lng: getLng(lang)}), yourResponseHref, changeLabel(lang)),
-        summaryRow(t('PAGES.CHECK_YOUR_ANSWER.WHEN_PAY', {lng: getLng(lang)}), getPaymentOption(claim, lang), paymentOptionHref, changeLabel(lang)),
-      ]);
+      if(claim.respondent1.responseType === ResponseType.FULL_ADMISSION) {
```
Comment (laperla11) [inline src/main/services/features/response/checkAnswers/responseSection/buildYourResponsePaymentSection.ts:65]: same here

**Review by laperla11 — APPROVED** [review]

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 1 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 2 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 3 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 4 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 5 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 6 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/checkAnswersService.ts (hunk 7 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/checkAnswersService.ts]: Added

File: src/main/services/features/response/checkAnswers/responseSection/buildYourResponsePaymentSection.ts (hunk 8 above)
Comment (RosDiaf) [reply src/main/services/features/response/checkAnswers/responseSection/buildYourResponsePaymentSection.ts]: Added

**Pull request merged**